### PR TITLE
add: Avatar Component

### PR DESCRIPTION
**Pull request by Tarandeep-s1ngh**
1. Folder Structure Created: Components -> avatar -> avatar.html & avatar.css
2. add: Avatar Component
3. Variations: 
- Profile Avatar
- Round Avatar
- Avatar of different sizes
4. add: Documentation page for Avatar Component

_gif attached_
Avatar-comp

### CODE REVIEW

File: Components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+.avatar {
+    width: 3rem;
+}
```
Comment (shobit1337) [inline Components/avatar/avatar.css:4]: I think It'll be better if we provide it with a fixed height too.
Otherwise, the avatar's height might stretch

**Review by muskaanshah — APPROVED** [review]
Looks good

**Review by mohammadkhizerkhan — APPROVED** [review]
looks good

**Review by shobit1337 — CHANGES_REQUESTED** [review]

File: Components/avatar/avatar.css (hunk 1 above)
Comment (Tarandeep-s1ngh) [reply Components/avatar/avatar.css:4]: Thanks Shobit, I'll look into it and make the required changes!

Comment (mohammadkhizerkhan) [reply Components/avatar/avatar.css:4]: if we don't use height then it will take default as "auto" which is fine for avatar.

**Pull request merged**